package.json

```
@@ -49,7 +49,8 @@
     "mock-android": "yarn android:mock && yarn android:install",
     "staging-android": "yarn android:staging && yarn android:install",
     "prettier": "prettier --write \"src/**/*.js\"",
-    "lint": "eslint src --fix --ext .js,.json",
+    "lint": "eslint src --ext .js,.jsx,.ts,.tsx,.json",
+    "lint:fix": "eslint src --fix --ext .js,.jsx,.ts,.tsx,.json",
```

You can avoid some duplication with that suggestion
```suggestion
    "lint:fix": "yarn lint --fix",
```